index.js

```diff
@@ -32,14 +33,20 @@ module.exports = async ({
     adapter: HFDBLowDBAdapter({ dbPath: algoDBPath })
   })
 
+  const logAlgoOpts = {
+    logAlgo: ALGO_LOG ? ALGO_LOG === 'true' : false,
+    logAlgoDir: ALGO_LOG_DIR || ''
```

Comment:
how about using a default here that points to `~/.honeyframework/algo-logs` or similar?

Reply:
https://github.com/bitfinexcom/bfx-hf-ui/blob/985b31931a3aa742ee018507c40d857cad537382/scripts/start-api-server.js#L15

Reply:
nevermind, lets add this with the HF UI setting for algo logs